traffic_incident_parser/launch/traffic_incident_parser.launch

```diff
@@ -0,0 +1,20 @@
+<!--
+  Copyright (C) 2020 LEIDOS.
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+  http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+-->
+<!--
+  This file is used to launch the traffic incident node
+-->
+
+<launch>
+  <node name="traffic_incident_parser" pkg="traffic_incident_parser" type="traffic_incident_parser_node">
+  </node>
+</launch>
```

Comment:
Should we invoke the launch file from higher launch file like guidance.launch (or wherever it should be called)? or is it not ready yet?

Reply:
not ready yet